examples/connect/src/app.tsx

```diff
@@ -0,0 +1,55 @@
+import {combineReducers, createStore} from 'redux';
+import {CheckBox, CheckBoxSelectEvent, Color, contentView, Stack} from 'tabris';
+import {DefaultRootState, injector, StateProvider} from 'tabris-decorators';
+import {register} from 'tabris-decorators';
+import {ExampleComponent} from './ExampleComponent';
+
+injector.jsxProcessor.unsafeBindings = 'error';
+
+declare module 'tabris-decorators' {
+
+  export interface DefaultRootState {
+    str: string;
+    num: number;
+  }
+
+}
+
+type RootState = DefaultRootState;
+
+type Actions = {
+  type: 'TOGGLE_VALUES',
+  checked: boolean
+};
+
+const store = createStore<RootState, Actions, unknown, unknown>(
+  combineReducers<RootState, Actions>({
+    num(state, action) {
+      if (action.type === 'TOGGLE_VALUES') {
+        return action.checked ? 90 : 10;
+      }
+      return state || 0;
+    },
+    str(state, action) {
+      if (action.type === 'TOGGLE_VALUES') {
+        return action.checked ? 'Another Hello World' : 'Hello World';
+      }
+      return state || 'Hello World';
```

Comment:
As a good practice, always use `??` to give defaults for unset values in TypeScript code? The state might be a valid falsy value.

Reply:
Okay, here I see it b.c. state could be an empty string. I'm just a bit concerned that it may not be universally understood since `??` is so new. I had to look it up myself to get the details right.